addons/website_event/controllers/main.py

```diff
@@ -312,7 +312,20 @@ def registration_new(self, event, **post):
                 availability_check = False
         if not tickets:
             return False
-        return request.env['ir.ui.view']._render_template("website_event.registration_attendee_details", {'tickets': tickets, 'event': event, 'availability_check': availability_check})
+        default_user = {}
+        if not request.env.user._is_public():
+            default_user = {
+                "name": request.env.user.name,
+                "email": request.env.user.email,
+                "phone": request.env.user.mobile or request.env.user.phone,
+            }
+        return request.env['ir.ui.view']._render_template("website_event.registration_attendee_details",
+            {
+                'tickets': tickets,
+                'event': event,
+                'availability_check': availability_check,
+                'user': default_user,
```

Comment:
Actually let's rename the dictionary key and the variable to "default_first_attendee".
So it makes more sense when used in the template.

Other than that it looks a lot better, thanks for your work!